<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sms_list</class>
 <widget class="QDialog" name="sms_list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>707</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>334</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>方正黑体_GBK</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Messages List</string>
  </property>
  <widget class="QPushButton" name="contactButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>182</y>
     <width>300</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>512</y>
     <width>218</width>
     <height>53</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>512</y>
     <width>84</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>118</y>
     <width>20</width>
     <height>64</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="phone_numberLable">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>145</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿等宽微米黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>610</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="timedSendRadio">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>150</y>
     <width>106</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Timed Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setTimeButton">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>120</y>
     <width>106</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>106</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>89</width>
     <height>27</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Timed Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLable">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>125</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>timedSendRadio</zorder>
  <zorder>setTimeButton</zorder>
  <zorder>contactButton</zorder>
  <zorder>listWidget</zorder>
  <zorder>textEdit</zorder>
  <zorder>sendButton</zorder>
  <zorder>line_2</zorder>
  <zorder>phone_numberLable</zorder>
  <zorder>pushButton</zorder>
  <zorder>nameLable</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>sms_list</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
